src/AppInstallerCLICore/Argument.cpp

```diff
@@ -63,6 +63,8 @@ namespace AppInstaller::CLI
             return Argument{ "msix", 'm', Args::Type::Msix, Resource::String::MsixArgumentDescription, ArgumentType::Flag };
         case Args::Type::ListVersions:
             return Argument{ "versions", NoAlias, Args::Type::ListVersions, Resource::String::VersionsArgumentDescription, ArgumentType::Flag };
+        case Args::Type::ListAll:

```

Comment:
nit: since this is for list command specifically, this could be moved to ListCommand.cpp, just like the UpgradeCommand. This Argument::ForType was initially including every argument, then we switched to be "Argument::ForType is for common args, and command specific ones could just be defined in the specific command.cpp". Btw, upgrade all does not have an alias, should this be the same for consistency?

Reply:
Fixed, and I removed the alias for consistency :)